joern-cli/build.sbt

```diff
@@ -29,6 +28,48 @@ topLevelDirectory := Some(packageName.value)
 
 Compile/packageDoc/mappings := Seq()
 
+lazy val c2cpg = project.in(file("frontends/c2cpg")).enablePlugins(JavaAppPackaging).settings(
+  libraryDependencies += "io.shiftleft" %% "c2cpg" % Versions.cpg,
+  Compile/mainClass := Some("io.shiftleft.c2cpg.C2Cpg"),
+)
+Universal/mappings ++= NativePackagerHelper.contentOf((c2cpg/stage).value).map {
+  case (file, name) => file -> s"frontends/c2cpg/$name"
+}
+
+lazy val fuzzyc2cpg = project.in(file("frontends/fuzzyc2cpg")).enablePlugins(JavaAppPackaging).settings(
+  libraryDependencies += "io.shiftleft" %% "fuzzyc2cpg" % Versions.cpg,
+  Compile/mainClass := Some("io.shiftleft.fuzzyc2cpg.FuzzyC2Cpg"),
+)
+Universal/mappings ++= NativePackagerHelper.contentOf((fuzzyc2cpg/stage).value).map {
+  case (file, name) => file -> s"frontends/fuzzyc2cpg/$name"
+}
+
+lazy val ghidra2cpg = project.in(file("frontends/ghidra2cpg")).enablePlugins(JavaAppPackaging).settings(
+  libraryDependencies += "io.joern" %% "ghidra2cpg" % Versions.ghidra2cpg,
+  Compile/mainClass := Some("io.joern.ghidra2cpg.Main"),
+)
+Universal/mappings ++= NativePackagerHelper.contentOf((ghidra2cpg/stage).value).map {
+  case (file, name) => (file, s"frontends/ghidra2cpg/$name")
+}
+
+lazy val js2cpg = project.in(file("frontends/js2cpg")).enablePlugins(JavaAppPackaging).settings(
+  libraryDependencies += "io.shiftleft" %% "js2cpg" % Versions.js2cpg,
+  Compile/mainClass := Some("io.shiftleft.js2cpg.core.Js2CpgMain"),
+)
+Universal/mappings ++= NativePackagerHelper.contentOf((js2cpg/stage).value).map {
+  case (file, name) => file -> s"frontends/js2cpg/$name"
+}
+
+lazy val plume = project.in(file("frontends/plume")).enablePlugins(JavaAppPackaging).settings(
+  libraryDependencies ++= Seq(
+    "io.github.plume-oss" % "plume" % "0.5.12",
```

Comment:
Fix for Plume's compatibility issues now in [0.5.13](https://github.com/plume-oss/plume/releases/tag/v0.5.13)